src/broken_crystals.cr

```diff
@@ -36,6 +36,11 @@ module BrokenCrystals
     render "src/views/main.ecr"
   end
 
+  get "/vulns" do |env|
+    env.response.headers["Content-Type"] = "text/html"
+    render "src/views/vulns.ecr"
+  end
+
   get "/vuln/greeter" do |_|
   end
```

Comment:
you should add `general` or `home` controller and just push all misc endpoints there like greet and shit

Reply:
![image](https://user-images.githubusercontent.com/19290620/81416887-aaf4d180-914a-11ea-9c02-7fda665dc7d5.png)
what